### DISCUSSION TITLE
Disable showing RTCP as the default setting

### DISCUSSION THREAD

**Discussion by Gasmanz** [sipcapture/homer#522] [Q&A]
Does anyone know if it's possible to disable showing the RTCP messages in the SIP Flow by default. Having to do this everytime I look at a SIP Flow is very annoying after the 100th time. For me the RTCP details are rarely used in call investigations. In fact it would be good to have the ability to set the defaults for any of the flow display parameters.

Thanks in advance.

**Comment by lmangani** [comment]
Your session settings should be retained across the same browser session, if that's not the case please open an issue instead.
This said, and for your information, to come and define a feature as "very annoying" is not going to motivate anyone ;)

**Reply by Gasmanz** [reply]
Every time I click on a new session ID from the same search results page I have to untick the RTCP option. Disabling showing the RTCP is not persistent in any way, even if I reopen the same session ID. If it can be defaulted to being displayed, can it be defaulted to not being displayed? Is it simply hardcoded to always displaying RTCP by default?

![image](https://user-images.githubusercontent.com/81398328/152088979-74289e89-5bde-435a-a0bc-6d889d39ce0e.png)

**Comment by lmangani** [comment]
This is a bug report and should be channeled through an issue.
I've created one, please move your activity there https://github.com/sipcapture/homer/issues/523
